Sources/PointFree/Account/Account.swift

```diff
@@ -74,29 +77,20 @@ private let profileRowView = View<Database.User> { currentUser in
           label([`class`([labelClass])], ["Name"]),
           input([
             `class`([blockInputClass]),
-            name(Database.User.CodingKeys.name.stringValue),
+            name(ProfileData.CodingKeys.name.stringValue),
```

Comment:
technically these input names should be based off of `ProfileData` cause that's the data type created from submitting.